docs/supported_instruments.rst

```diff
@@ -106,6 +106,14 @@ ISS FPMU
 .. automodule:: pysatNASA.instruments.iss_fpmu
    :members:
 
+.. _jpl_gps:
```

Comment:
I tried to double check this via RTD but normal pulls aren't built. A setting was changed over in pysat to build more which can be helpful... Not a requirement.

Reply:
Documented in #60